Core/Core/CourseSync/CourseSyncDownloader/Model/CourseSyncFilesInteractor.swift

```diff
@@ -94,4 +213,40 @@ public final class CourseSyncFilesInteractorLive: CourseSyncFilesInteractor, Loc
             .eraseToAnyPublisher()
         }
     }
+
+    public func removeUnavailableFiles(
+        courseId: String,
+        newFileIDs: [String]
+    ) -> AnyPublisher<Void, Error> {
+        guard let sessionID = env.currentSession?.uniqueID else {
+            return Fail(error:
+                NSError.instructureError(
+                    NSLocalizedString(
+                        "There was an unexpected error. Please try again.",
+                        bundle: .core,
+                        comment: ""
+                    )
+                )
+            )
+            .eraseToAnyPublisher()
+        }
+
+        let courseFolderURL = URL.Directories.documents.appendingPathComponent("\(sessionID)/Offline/Files/course-\(courseId)")
```

Comment:
I think we should move all these directory factories to the URL.Directories namespace. It's easy now to update the url in one place and forget about the other instances.